public/js/components/EditorBreakpoint.js

```diff
@@ -1,25 +1,21 @@
 const React = require("react");
+const ReactDOM = require("react-dom");
+
 const { PropTypes } = React;
 const classnames = require("classnames");
+const Svg = require("./utils/Svg");
+
+let breakpointSvg = document.createElement("div");
+ReactDOM.render(Svg("breakpoint"), breakpointSvg);
```

Comment:
yeah - that's a good point, maybe it makes sense to add an exported function to Svg.js for a raw node as well